pkg/collector/corechecks/snmp/snmp_test.go

```diff
@@ -1629,31 +1642,49 @@ tags:
 
 	// language=json
 	event := []byte(fmt.Sprintf(`
-{
-  "subnet": "127.0.0.0/30",
-  "namespace":"default",
-  "devices": [
-    {
-      "id": "default:1.2.3.5",
-      "id_tags": [
-        "device_namespace:default",
-        "snmp_device:1.2.3.5"
-      ],
-      "tags": [
-        "agent_version:%s",
-        "autodiscovery_subnet:127.0.0.0/30",
-        "device_namespace:default",
-        "mytag:val1",
-        "snmp_device:1.2.3.5"
+	{
```

Comment:
Shall we keep the same indent as before to reduce changed lines? Makes it easier to review also.

Reply:
Good catch, should be fixed thanks!

Reply:
its not, taking another look 😅